server/upstream/upstream-tarballs.txt

```diff
@@ -39,6 +39,12 @@
 #     https://svn.revolutionlinux.com/MILLE/XTERM/trunk/libflashsupport/Tarballs/
 #   + the info field should be the URL of the svn directory
 #
+# github::
+#   + use this when the tarballs are hosted on github
+#   + all tags that match accepted version schema are considered
+#   + works for projects that do not do the github-style releases
+#   + the expected url schema is https://github.com/project/repository/releases
```

Comment:
So I would change the syntax in order to not pass an URL, but just the project and the repo -- after all, we can create the github URL (and this makes changing the way we find the tarballs, in case it moves to another page, easier)